### PR TITLE
simplified doc

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

**Review by bruth — APPROVED** [review]
LGTM.

File: jetstream.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ associated with a stream.
 
 Messages are replayed from a stream by _consumers_. A consumer configuration
 specifies which messages should be presented. For example a consumer may only be
-interested in viewing messages from a specific sequence or since a specific
-time. The configuration also specifies if the server should require for a
-message to be acknowledged, and how long to wait for acknowledgements. The
+interested in viewing messages from a specific sequence or starting from a
+specific time. The configuration also specifies if the server should require for
+a message to be acknowledged, and how long to wait for acknowledgements. The
```
[the remark this answers is not in the document]
Comment (aricart) [reply jetstream.md]: fixed

File: jetstream.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ const stream = "mystream";
 const subj = `mystream.*`;
 await jsm.streams.add({ name: stream, subjects: [subj] });
 
-// publish a reg nats message directly to the stream
+// jetstream can capture nats core messages
```
[the remark this answers is not in the document]
Comment (aricart) [reply jetstream.md:50]: fixed

File: jetstream.md [hunk 3]
```diff
@@ -107,20 +107,20 @@ await jsm.consumers.delete(stream, "me");
 
 ## JetStream Client
 
-The JetStream client functionality presents a JetStream view on a NATS client.
-While you can implement your own API to interact with streams, the JetStream
-APIs make this convenient.
+The JetStream client presents an API for working with messages stored on a
```
[the remark this answers is not in the document]
Comment (aricart) [reply jetstream.md:110]: fixed

File: jetstream.md [hunk 4]
```diff
@@ -107,20 +107,20 @@ await jsm.consumers.delete(stream, "me");
...
 await jsm.streams.add({ name: "a", subjects: ["a.*"] });
 
 // create a jetstream client:
 const js = nc.jetstream();
 
-// to publish messages to a stream:
+// to publish messages to a stream
 let pa = await js.publish("a.b");
-// the jetstream returns an acknowledgement with the
+// jetstream returns an acknowledgement with the
```
[the remark this answers is not in the document]
Comment (aricart) [reply jetstream.md:123]: fixed

File: jetstream.md [hunk 5]
```diff
@@ -130,8 +130,9 @@ const duplicate = pa.duplicate;
 // More interesting is the ability to prevent duplicates
 // on messages that are stored in the server. If
 // you assign a message ID, the server will keep looking
-// for the same ID for a configured amount of time, and
-// reject messages that sport the same ID:
+// for the same ID for a configured amount of time (specified
+// by the stream configuration), and reject messages that
+// sport the same ID:
```
[the remark this answers is not in the document]
Comment (aricart) [reply jetstream.md:135]: fixed

File: examples/jetstream/simplified/06_heartbeats.ts [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+const c = await js.consumers.get(stream, consumer);
+while (true) {
+  const messages = await c.consume({ max_messages: 1 });
+
+  // watch the to see if the consume operation misses heartbeats
+  (async () => {
+    for await (const s of await messages.status()) {
+      if (s.type === ConsumerEvents.HeartbeatsMissed) {
+        // you can decide how many heartbeats you are willing to miss
```
[the remark this answers is not in the document]
Comment (aricart) [reply examples/jetstream/simplified/06_heartbeats.ts]: The server should be sending heartbeats whenever messages are not being sent. This allows the client to know that the server request is alive.

File: examples/jetstream/simplified/07_consume_jobs.ts [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+const nc = await connect();
+
+// make a stream and fill with messages, and create a consumer
+const { stream, consumer } = await newTestStream(nc, 100);
+
+// retrieve an existing consumer
+const js = nc.jetstream();
+const c = await js.consumers.get(stream, consumer);
+
+// this example uses a consume that is monitors a rate limiter
```
[the remark this answers is not in the document]
Comment (aricart) [reply examples/jetstream/simplified/07_consume_jobs.ts]: Changed to `// this example controls parallel processing of the messages
// by only allowing 5 concurrent messages to be processed
// and then only allowing additional processing as others complete`

**Pull request merged**